lib/config.js

```diff
@@ -23,38 +23,45 @@ const Config = {
       default: true,
       order: 0
     },
-    showAutocompleteFirst: {
-      title: "Show Hydrogen's autocomplete suggestions first",
-      includeTitle: false,
+    autocompleteSuggestionPriority: {
+      title: "Autocomple Suggestion Priority",
+      description:
+        "Specify the sort order of Hydrogen's autocomplete suggestions.\nNote the default providers like snippets have priority of 1.",
+      type: "integer",
+      default: 3,
+      order: 1
+    },
+    showInspectorResultsInAutocomplete: {
+      title: "Enable Autocomplete description",
```

Comment:
Maybe instead `Enhance Autocomplete Description` also maybe explain that disabled will still provide autocomplete results (even though this is self explanatory, it can help newer coders/users)

Reply:
Yeah good point, thanks ;)

Reply:
Addressed.